packages/esm-patient-banner-app/src/contact-details/contact-details.scss

```diff
@@ -18,7 +18,8 @@
 
 .row .col {
   flex: 1;
-  padding: 1.3125rem;
+  padding: 1.0125rem;
+  padding-top: 0.15rem;
```

Comment:
You probably want to condense this into one declaration using the padding property: `padding: padding-top-value padding-right-value padding-bottom-value padding-left-value` . As currently written, it's not entirely clear what amounts of padding should be applied.

Reply:
Ok, I had used rem to strive for uniformity but I will update this.